README.md

```diff
@@ -1 +1,68 @@
-# module-ballerina-config
\ No newline at end of file
+Ballerina Config Library
+=========================
+
+[![Build](https://github.com/ballerina-platform/module-ballerina-config/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-config/actions?query=workflow%3ABuild)
+[![Daily build](https://github.com/ballerina-platform/module-ballerina-config/workflows/Daily%20build/badge.svg?branch=master)](https://github.com/ballerina-platform/module-ballerina-config/actions?query=workflow%3A%22Daily+build)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-config.svg)](https://github.com/ballerina-platform/module-ballerina-config/commits/master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The config library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> 
+Ballerina</a> language.
+
+This module provides the interface and functionality to read configurations from environment variables, TOML files, 
+and command-line parameters and build a consolidated set of configurations.
+
+For more information go to [The Config Module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/config/).
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/config-api.html).
+
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+Download and install Java SE Development Kit (JDK) version 8 (from one of the following locations).
+
+   * [Oracle](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
+   
+   * [OpenJDK](http://openjdk.java.net/install/index.html)
+   
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+     
+### Building the Source
+
+Execute the commands below to build from source.
+
+1. To build the library:
+        
+        ./gradlew clean build
+
+2. To run the integration tests:
+
+        ./gradlew clean test
```

Comment:
This is command working? From what I can see there is no test task in the build

Reply:
Updated build commands in cf97941606aa4fef8da89f03dc11fb2aaeb5f3ea